src/object/srv_obj_migrate.c

```diff
@@ -2471,7 +2471,8 @@ migrate_one_epoch_object(daos_epoch_range_t *epr, struct migrate_pool_tls *tls,
 			if (rc != -DER_INPROGRESS) {
 				daos_anchor_set_flags(&dkey_anchor,
 						      DIOF_WITH_SPEC_EPOCH |
-						      DIOF_TO_SPEC_GROUP);
+						      DIOF_TO_SPEC_GROUP |
```

Comment:
just confirm why originally with DIOF_TO_SPEC_GROUP flag?

Reply:
because object enumeration will be limited to the rebuild group only.